src/frontend/src/components/Charge/index.tsx

```diff
@@ -20,36 +20,31 @@ export default class Charge extends React.Component<Props> {
     } = this.props.charge;
 
 
-    const knownDisposition = (disposition: any, date: any) => {
-      let dispositionEvent = disposition.status;
-      if (disposition.status === "Convicted") {
-        dispositionEvent += " - " + disposition.date;
-      } else if (disposition.status === "Unrecognized") {
-        dispositionEvent += " (\"" + disposition.ruling + "\")";
-      }
-      if (disposition.amended) {
-        dispositionEvent += " (Amended)"
+    const dispositionEvent = (disposition: any, date: any) => {
+      let dispositionEvent;
+      if (disposition === null) {
+        dispositionEvent = "Missing";
+      }  else {
+        dispositionEvent = disposition.status;
+        if (disposition.status === "Convicted") {
```

Comment:
What about when status is acquitted?

Reply:
The "Dismissed" ruling is being assigned at the line just above this, `dispositionEvent = disposition.status;` and that should be the entire `dispositionEvent`. We exclude the dismissal date if the charge is dismissed because it has no purpose for eligibility of or applying for expungement.